test/e2e/puppeteer.js

```diff
@@ -124,6 +124,7 @@ const exceptionList = [
 
 	// WebGPURenderer: Unknown problem
 	'webgpu_postprocessing_afterimage',
+	'webgpu_postprocessing_3dlut',
```

Comment:
The reason I added the previous builds was because it was failing in e2e. So exceptions would not be necessary. One way to solve this would be to split the PR between API and examples, I'm not sure if puppeteer did the builds at test time in `three.module.js`.

Reply:
> I'm not sure if puppeteer did the builds at test time in three.module.js?

The CI executes a build here before running the E2E tests:

https://github.com/mrdoob/three.js/blob/1e4f2b16a098dcae181232a854da2832675ae75c/.github/workflows/ci.yml#L87

But it seems the command only updates `three.module.js`.

Reply:
Maybe if we could add it `rollup.config.js`?

```js
export default ( args ) => args.configOnlyModule ? [ builds[ 0 ], builds[ 3 ] ] : builds;
```

https://github.com/mrdoob/three.js/blob/1e4f2b16a098dcae181232a854da2832675ae75c/utils/build/rollup.config.js#L133

Reply:
Sounds good!